README.rst

```diff
@@ -21,11 +21,19 @@ package:
     /usr/bin/make
     which cmake
     /usr/bin/cmake
+    which git
+    /usr/bin/git
 
 .. code:: sh
 
     pip install h3
 
+Installing on Windows
+---------------------
+
+On Windows you will also need to have ``bash`` (for example from Git)
```

Comment:
`bash` specifically? Or some shell?

Reply:
`setup.py` specifically tries to invoke `bash`, it's possible another *nix shell will work, but it will need to alias itself to `bash`.